bciers/apps/administration/app/bceidbusiness/industry_user/operations/[operationId]/facilities/[facilityId]/page.tsx

```diff
@@ -1,12 +1,15 @@
 import { UUID } from "crypto";
-import Facility from "@/administration/app/components/facilities/Facility";
+import FacilityPage from "@/administration/app/components/facilities/FacilityPage";
```

Comment:
I may be remembering this wrong, but I thought we'd decided to not have separate routes for edit vs. add? I see separate page.tsx in this PR

Reply:
We did discuss this, in the end I landed on using the separate routes for the sake of clarity over sake of dry...I hope you agree ;)

Reply:
Works for me ,FYI @marcellmueller. We've got just [operation] for the register workflow so I'll do something about that in the first step components

Reply:
Sounds good to me, thanks for the update!